pkgs/by-name/up/uplift/package.nix

```
@@ -0,0 +1,63 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  nix-update-script,
+}:
+
+buildGoModule (finalAttrs: {
+  pname = "uplift";
+  version = "2.26.0";
+
+  src = fetchFromGitHub {
+    owner = "gembaadvantage";
+    repo = "uplift";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-cXZAxsQWjzYgoRAEKflJFcVN0OabdPHuJonJ0KMEaXs=";
+  };
+
+  vendorHash = "sha256-mjhNgy/QGHVdujmy94NQaoyvfrVmsIKZS3uWYK1XQr0=";
+
+  ldflags = with finalAttrs.src; [
+    "-s"
+    "-w"
+    "-X=github.com/gembaadvantage/uplift/internal/version.version=${tag}"
+    "-X=github.com/gembaadvantage/uplift/internal/version.gitCommit=${tag}"
+    "-X=github.com/gembaadvantage/uplift/internal/version.gitBranch=main"
+    "-X=github.com/gembaadvantage/uplift/internal/version.buildDate=1970-01-01T00:00:00Z"
+  ];
+
+  doCheck = false;
+
+  doInstallCheck = true;
+
+  installCheckPhase = with finalAttrs; ''
+    runHook preInstallCheck
+
+    $out/bin/${meta.mainProgram} version | grep ${version}
+
+    runHook postInstallCheck
+  '';
+
+  passthru.updateScript = nix-update-script { };
+
+  meta = {
+    description = "Semantic versioning the easy way";
+    longDescripton = ''
```

Breakdown of the CI failure:
```suggestion
    longDescription = ''
```